### PR TITLE
swiftformat: new port

### PR DESCRIPTION
**Pull request by jrabinow**
#### Description

Add new port: swiftformat

###### Tested on
macOS 14.6.1 23G93 arm64
Xcode 15.4 15F31d

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [x] checked your Portfile with `port lint --nitpick`? `Error: Using openmaintainer without any other maintainer` who should I add
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`? `-vs` since trace mode is broken on arm64
- [x] tested basic functionality of all binary files?
- [] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken? N/A

### CODE REVIEW

File: devel/swiftformat/Portfile
Comment (markemer) [inline devel/swiftformat/Portfile:13]: This I guess should be `nomaintainer`? What's the rule for new ports, usually I've only ever seen us add `nomaintainer` to old ports.

Comment (reneeotten) [reply devel/swiftformat/Portfile:13]: you can submit a port with oh t a maintainer, but given the that you presumably care about something you submit it would be great if the submitter would volunteer to maintain it.

But indeed, "openmaintainer" is incorrect as that only signifies that others -besides the listed maintainer- can make changes without requiring approval.

Comment (jrabinow) [reply devel/swiftformat/Portfile]: Thanks for the feedback. Fixed!

File: devel/swiftformat/Portfile
Comment (reneeotten) [inline devel/swiftformat/Portfile:13]: the first entry here should be your (obfuscated) e-mail address; what you have now implies a ....@macports.org address which isn't correct

**Review by reneeotten — CHANGES_REQUESTED** [review]

File: devel/swiftformat/Portfile
Comment (jrabinow) [reply devel/swiftformat/Portfile]: Sorry about that. I went and looked up [the guide](https://guide.macports.org/#reference.keywords.maintainers) to make sure I got it right this time.

Judging by the output of `rg maintainers|grep -v nomaintainer|grep -E '\{@[^ ]*\}'`, there aren't many ports without email, and so providing an email address is preferable. I'm wondering if it's nevertheless accepted to not provide an email address

Comment (markemer) [reply devel/swiftformat/Portfile]: Pre-github that was the only way - I don't know what the policy is now. I know we'd prefer a email, as we mostly discuss over mailing list.

**Pull request merged**
